### PR TITLE
chore: release preparation for 1.0.0 rc.0

### PR DESCRIPTION
**Pull request by tolerants**
- Removed forked lerna version
- Added release rc workflow
- Added auto-creation of github release
- Removed lerna configuration for apps

closes: [HRZ-3171](https://spryker.atlassian.net/browse/HRZ-3171), [HRZ-3173](https://spryker.atlassian.net/browse/HRZ-3173), [HRZ-3172](https://spryker.atlassian.net/browse/HRZ-3172)

[HRZ-3171]: https://spryker.atlassian.net/browse/HRZ-3171?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[HRZ-3173]: https://spryker.atlassian.net/browse/HRZ-3173?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[HRZ-3172]: https://spryker.atlassian.net/browse/HRZ-3172?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .github/workflows/release-rc.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          npm whoami
+      - name: Auth Git
+        run: |
+          git config user.email "${{ env.GIT_EMAIL }}"
+          git config user.name "${{ env.GIT_USER }}"
+      - name: Install Deps
+        run: npm ci
+      - name: Build packages
+        run: npx nx run-many --target=build --all --parallel=2 --exclude testfront,storybook,fulfillment,storefront
+      - name: Fix js extension
```
Comment (AndreyYevtukhov) [inline .github/workflows/release-rc.yml:45]: Do we need this step? It was removed in the test workflow or am I wrong?

File: .github/workflows/release-rc.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
+name: Release rc Workflow
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run && github.event.workflow_run.conclusion == 'success' || true }}
+    environment: 'Release NPM'
```
Comment (AndreyYevtukhov) [inline .github/workflows/release-rc.yml:10]: It seems like this name will appear in Jira - https://spryker.atlassian.net/jira/software/c/projects/HRZ/deployments?startDate=-3m&endDate=now

but I wonder how to mark it as a production one?

File: .github/workflows/release-rc.yml (hunk 1 above)
Comment (tolerants) [reply .github/workflows/release-rc.yml:45]: It's needed for real release.

File: .github/workflows/release-rc.yml (hunk 2 above)
Comment (tolerants) [reply .github/workflows/release-rc.yml:10]: But it's ok that all release workflows are marked as `Release NPM`, no? Or you want to find a better name?

**Review by AndreyYevtukhov — APPROVED** [review]

File: .github/workflows/release-rc.yml (hunk 2 above)
Comment (AndreyYevtukhov) [reply .github/workflows/release-rc.yml:10]: name is OK, I just wonder how it can be marked as production release :)

**Pull request merged**